src/page/sign-in-step.page.ts

```diff
@@ -1,14 +1,14 @@
-import { $, ElementFinder, browser, ExpectedConditions } from 'protractor';
+import { ElementFinder, browser, ExpectedConditions, by, element } from 'protractor';
 
 export class SignInCardStepPage {
   private emailField: ElementFinder;
   private passwordField: ElementFinder;
   private signInOption: ElementFinder;
 
   constructor() {
-    this.emailField = $('#email');
-    this.passwordField = $('#passwd');
-    this.signInOption = $('#SubmitLogin > span');
+    this.emailField = element(by.id('email'));
```

Comment:
these are good locators but the same that above, try to use element or $() but avoid to merge them